src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/DockerProcessor.kt

```diff
@@ -0,0 +1,43 @@
+/**
+    Copyright 2018 BarD Software s.r.o
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Author: Mikhail Shavkunov (@shavkunov)
+ */
+package com.bardsoftware.backend.fleet.rmarkdown
+
+import com.spotify.docker.client.DefaultDockerClient
+import com.spotify.docker.client.DockerClient
+import com.spotify.docker.client.messages.ContainerConfig
+
+class DockerProcessor {
+    private val docker: DockerClient = DefaultDockerClient.fromEnv().build()
+    private var containerId: String? = null
+
+    fun getMd5Sum(message: String): String {
+        val quotedMessage = "\"$message\""
+
+        val containerConfig = ContainerConfig.builder()
+                .image("busybox")
+                .cmd("sh", "-c", "md5sum", "<<<", quotedMessage)
+                .build()
+
+        val creation = docker.createContainer(containerConfig)
+        containerId = creation.id()
```

Comment:
Why do you need `containerId` as a member variable?

Also, we don't need to kill containers or stop the client, but we do need to remove container, otherwise your docker will be fully packed with containers pretty soon.

Reply:
Done